uint32 seconds           # ROS time the message was sent in seconds
uint32 millis      # ROS time the message was sent, millisecond component
float32 motor1          # Speed of Motor1 in radians per second
float32 motor2          # Speed of Motor2 in radians per second
float32 motor3          # Speed of Motor3 in radians per second
float32 motor4          # Speed of Motor4 in radians per second